README.md

```diff
@@ -22,6 +19,11 @@ We consider this library pretty stable and use it in many critical production sy
 * [Code of Conduct](#code-of-conduct)
 * [User Manual](https://github.com/spotify/docker-client/blob/master/docs/user_manual.md)
 
+## Version compatibility
+docker-client is built and tested against the six most recent minor releases of Docker.
+Right now these are 1.9.1 - 17.03.0-ce (specifically the ones [here][1]).
+We upload the artifact tested on Docker 1.12.1.
+See [Docker docs on the mapping between Docker version and API version][3].
```

Comment:
Update the link. The mapping is now here. https://docs.docker.com/engine/api/v1.27/#section/Versioning